shell/platform/windows/flutter_platform_node_delegate_windows.cc

```diff
@@ -33,14 +35,15 @@ void FlutterPlatformNodeDelegateWindows::Init(std::weak_ptr<OwnerBridge> bridge,
                                               ui::AXNode* node) {
   FlutterPlatformNodeDelegate::Init(bridge, node);
   ax_platform_node_ = ui::AXPlatformNode::Create(this);
-  assert(ax_platform_node_);
+  FML_DCHECK(ax_platform_node_) << "Failed to create AXPlatformNode";
 }
 
 // |ui::AXPlatformNodeDelegate|
 gfx::NativeViewAccessible
 FlutterPlatformNodeDelegateWindows::GetNativeViewAccessible() {
-  assert(ax_platform_node_);
-  return ax_platform_node_->GetNativeViewAccessible();
+  FML_DCHECK(ax_platform_node_)
+      << "AXPlatformNode hasn't been created;
+         return ax_platform_node_->GetNativeViewAccessible();
```

Comment:
This looks weird. Perhaps:

```suggestion
      << "AXPlatformNode hasn't been created";
  return ax_platform_node_->GetNativeViewAccessible();
```

Reply:
That's what I get for sending a patch from my Mac! Nice catch!